atlp-rwanda/ecommerce-app-champions-fe
# #78Feature: seller should be able to view a specific item

### what does this PR do?

- implement the frontend for a seller to view production in his/her collection
- implement the frontend for a seller to view specific items

### Description 

- implement the products component
- implement single product component

### How to manually test it

- clone the repo's URL
- checkout to ft-seller-view-products-cp-77
- run npm run dev
- navigate to routes
- navigate to ypur browser and click on vendors
- make sure that you are logged in as vendor
- click on  an eye icon to view details of the product
